src/debug_menu.cpp

```diff
@@ -61,6 +76,124 @@ class mission_debug
         static std::string describe( const mission &m );
 };
 
+static int player_uilist()
+{
+    const std::vector<uilist_entry> uilist_initializer = {
+        uilist_entry( DEBUG_MUTATE, true, 'M', _( "Mutate" ) ),
+        uilist_entry( DEBUG_CHANGE_SKILLS, true, 's', _( "Change all skills" ) ),
+        uilist_entry( DEBUG_LEARN_MA, true, 'l', _( "Learn all melee styles" ) ),
+        uilist_entry( DEBUG_UNLOCK_RECIPES, true, 'r', _( "Unlock all recipes" ) ),
+        uilist_entry( DEBUG_EDIT_PLAYER, true, 'p', _( "Edit player/NPC" ) ),
+        uilist_entry( DEBUG_DAMAGE_SELF, true, 'd', _( "Damage Self" ) ),
+        uilist_entry( DEBUG_SET_AUTOMOVE, true, 'a', _( "Set automove route" ) )
+    };
+
+    return uilist( _( "Info..." ), uilist_initializer );
+}
+
+static int info_uilist()
+{
+    const std::vector<uilist_entry> uilist_initializer = {
+        uilist_entry( DEBUG_GAME_STATE, true, 'g', _( "Check game state" ) ),
+        uilist_entry( DEBUG_DISPLAY_HORDES, true, 'h', _( "Display hordes" ) ),
+        uilist_entry( DEBUG_TEST_IT_GROUP, true, 'i', _( "Test Item Group" ) ),
+        uilist_entry( DEBUG_SHOW_SOUND, true, 's', _( "Show Sound Clustering" ) ),
+        uilist_entry( DEBUG_DISPLAY_WEATHER, true, 'w', _( "Display weather" ) ),
+        uilist_entry( DEBUG_DISPLAY_SCENTS, true, 'S', _( "Display overmap scents" ) ),
+        uilist_entry( DEBUG_SHOW_MUT_CAT, true, 'm', _( "Show mutation category levels" ) ),
+        uilist_entry( DEBUG_BENCHMARK, true, 'b', _( "Draw benchmark (X seconds)" ) ),
+        uilist_entry( DEBUG_TRAIT_GROUP, true, 't', _( "Test trait group" ) ),
+        uilist_entry( DEBUG_SHOW_MSG, true, 'd', _( "Show debug message" ) ),
+        uilist_entry( DEBUG_CRASH_GAME, true, 'C', _( "Crash game (test crash handling)" ) ),
+        uilist_entry( DEBUG_DISPLAY_NPC_PATH, true, 'n', _( "Toggle NPC pathfinding on map" ) )
+    };
+
+    return uilist( _( "Info..." ), uilist_initializer );
+}
+
+static int teleport_uilist()
+{
+    const std::vector<uilist_entry> uilist_initializer = {
+        uilist_entry( DEBUG_SHORT_TELEPORT, true, 's', _( "Teleport - Short Range" ) ),
+        uilist_entry( DEBUG_LONG_TELEPORT, true, 'l', _( "Teleport - Long Range" ) ),
+        uilist_entry( DEBUG_OM_TELEPORT, true, 'o', _( "Teleport - Adjacent overmap" ) )
+    };
+
+    return uilist( _( "Teleport..." ), uilist_initializer );
+}
+
+static int spawning_uilist()
+{
+    const std::vector<uilist_entry> uilist_initializer = {
+        uilist_entry( DEBUG_WISH, true, 'w', _( "Wish for an item" ) ),
```

Comment:
```suggestion
        uilist_entry( DEBUG_WISH, true, 'w', _( "Spawn an item" ) ),
```
On your consideration.